components/overflow-group/overflow-group.js

```diff
@@ -67,11 +67,14 @@ function createMenuItemMenu(node) {
 		||  node.querySelector('d2l-dropdown-button-subtle');
 
 	const openerText = node.text || menuOpener.text;
+	const disabled = node.disabled;
```

Comment:
For things that aren't inherently disable-able, this could be `undefined` I think. Maybe `!!node.disabled`?